docker-compose.yml

```diff
@@ -27,26 +27,26 @@ services:
       - ./docker/postgres_scram/init/:/docker-entrypoint-initdb.d/
     ports:
       - "6002:5432"
-  postgres_invalid_tls:
```

Comment:
Please don't remove the `postgres_invalid_tls` test, I'd like to keep tests for both scenarios

Reply:
I didn't remove the test. I just renamed the docker compose entry, because there is nothing inherently "invalid" about it. It only becomes "invalid" when the client does not trust the CA.

Reply:
You are right sorry, the original tests can simply not pass the cert option